src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -462,20 +462,33 @@ function untransformACL(mongoObject) {
 // cannot perform a transformation
 function CannotTransform() {}
 
+const transformInteriorAtom = atom => {
+  // TODO: check validity harder for the __type-defined types
+  if (typeof atom === 'object' && atom && !(atom instanceof Date) && atom.__type === 'Pointer') {
```

Comment:
With that change then every caller of `transformInteriorAtom` would have to be changed to check for `CannotTransform`. If tests still pass after removing the throw completely, I would rather just do that, as I'm pretty sure there is no way for that value to be a function or symbol anyway.